cmds/exp/pox/pox.go

```diff
@@ -277,21 +295,46 @@ func poxRun(args []string) error {
 	return nil
 }
 
+func extraMounts() error {
+	if *extra == "" {
+		return nil
+	}
+	v("Extra: %q", *extra)
+	// We have to specify the extra directories and do the create here b/c it is a squashfs. Sorry.
+	for _, e := range strings.Split(*extra, ",") {
+		m := mp{flags: mount.MS_BIND, perm: 0755}
+		mp := strings.Split(e, ":")
+		switch len(mp) {
+		case 1:
+			m.source, m.target = mp[0], mp[0]
+		case 2:
+			m.source, m.target = mp[0], mp[1]
+		default:
+			return fmt.Errorf("-extra: argument (%v) is not in the form src:target", mp)
+		}
+		v("Extra: append %q to chrootMounts", m)
+		chrootMounts = append(chrootMounts, m)
+	}
+	return nil
+}
+
 func pox() error {
 	flag.Parse()
 	if *debug {
 		v = log.Printf
 	}
-	if (*create && *run) || (!*create && !*run) {
-		return fmt.Errorf(usage)
+	if err := extraMounts(); err != nil {
+		return err
 	}
 	if *create {
-		return poxCreate(flag.Args())
+		if err := poxCreate(flag.Args()); err != nil {
+			return err
+		}
 	}
 	if *run {
 		return poxRun(flag.Args())
 	}
-	return fmt.Errorf(usage)
+	return nil
```

Comment:
so if you neither create nor run, what do you get?  looks like a silent exit.

overall, right now i think you have any combination of -e, -c, and -r is OK.